templates/corporate/attribution.html

```diff
@@ -23,6 +23,11 @@ <h1>Website attributions</h1>
                     <img alt="" src="/static/images/landing-page/companies/software-engineer.svg" />
                     <p>"<a href="https://iconscout.com/illustration/software-engineer-2043023">Software engineer Illustration</a>" By <a href="https://iconscout.com/contributors/delesign/illustrations">Delesign Graphic</a> is licensed under <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>
                 </li>
+                <li>
+                    <b>Default Integration Logo</b>
```

Comment:
Let's use sentence case here, as we generally do at Zulip, and match the formatting above of the other attribution label. I.e.:

> Default integration logo: